archunit/src/main/java/com/tngtech/archunit/lang/syntax/elements/FieldsShould.java

```diff
@@ -44,6 +44,26 @@
     @PublicAPI(usage = ACCESS)
     CONJUNCTION haveRawType(Class<?> type);
 
+    /**
+     * Asserts that fields not have a certain raw type.
```

Comment:
Maybe _"Asserts that fields **do** not have a certain raw type."_ ?
(Same below.)

Reply:
Good idea!